L1Trigger/L1TMuonBarrel/src/L1TMuonBarrelParamsHelper.cc

```diff
@@ -35,8 +35,8 @@ void L1TMuonBarrelParamsHelper::configFromPy(std::map<std::string, int>& allInts
 
 	setAssLUTPath(AssLUTpath);
 	///Read Pt assignment Luts
-	std::vector<LUT> pta_lut(0); pta_lut.reserve(19);
-	std::vector<int> pta_threshold(6); pta_threshold.reserve(10);
+	std::vector<LUT> pta_lut(19);
```

Comment:
@davidlange6  Yes, you are right.  Pushing back on pta_lut vector will place beyond index 18, 
which is not good.  Changing back to reserve for pta_lut.